scripts/dev-bundle-tool-package.js

```diff
@@ -25,35 +25,38 @@ var packageJson = {
     "babel-runtime": "7.0.0-beta.3",
     "@types/underscore": "1.11.15",
     underscore: "1.13.7",
-    "source-map-support": "https://github.com/meteor/node-source-map-support/tarball/1912478769d76e5df4c365e147f25896aee6375e",
+    "source-map-support":
+      "https://github.com/meteor/node-source-map-support/tarball/1912478769d76e5df4c365e147f25896aee6375e",
     "@types/semver": "7.5.8",
     semver: "7.6.3",
-    request: "2.88.2",
-    uuid: "3.4.0",
+    axios: "1.6.8",
```

Comment:
1. I see you removed the replaced request by axios, but I can't see where axios is being used 
2. Since you pretend to remove the request, could you use fetch instead of axios? it will be -1 dependency to care